cmd/things/main.go

```diff
@@ -157,21 +165,25 @@ func main() {
 	}
 }
 
-func newService(db *sqlx.DB, auth upolicies.AuthServiceClient, cacheClient *redis.Client, tracer trace.Tracer, logger mflog.Logger) (clients.Service, groups.Service, tpolicies.Service) {
+func newService(db *sqlx.DB, auth upolicies.AuthServiceClient, cacheClient *redis.Client, esClient *redis.Client, tracer trace.Tracer, logger mflog.Logger) (clients.Service, groups.Service, tpolicies.Service) {
 	database := postgres.NewDatabase(db, tracer)
 	cRepo := cpostgres.NewRepository(database)
 	gRepo := gpostgres.NewRepository(database)
 	pRepo := ppostgres.NewRepository(database)
 
 	idp := uuid.New()
 
-	policyCache := redischcache.NewCache(cacheClient)
+	policyCache := redispcache.NewCache(cacheClient)
```

Comment:
It is still used in `things.Idenitify` for mqtt

Reply:
Do we need identify since it is only used in `mqtt` to check is the `username` is the same as the `id`?
https://github.com/mainflux/mainflux/blob/72e18016901e26245dbc4f5cbe6646a315de1e74/mqtt/handler.go#LL87C1-L95C1

Reply:
Let's keep it for the time being.